### DISCUSSION TITLE
Using mannequin to model skydiving formation

### DISCUSSION THREAD

**Discussion by xGitter** [boytchev/mannequin.js#12] [Q&A]
Amazing Javascript library you created here.

It's a perfect match for building skydiving formation postures.

In the formations we usually have some standard body positions (I already modeled a few in the posture editor), for example see the current so called dive pool [1].

Do you have any tips on how to establish some workflow, in order to easily model new formations, without too much programming involved? I'm able to program, but I would like to allow other people to use it too.

I was thinking about pre-modeling most of the relevant standard postures, to have a toolset, to put these together. I have no idea how to put four people together yet. And in the long run, I would like to script it in some way, to be able to add another person etc.

Any hints would be appreciated.

[1] https://axis.center/pdf/drawgen/DivePool_VFS_4Way_USPA.pdf

**Comment by boytchev** [comment]
Hi -

Thanks for considering mannequin.js for demonstrating skydiving formations.

Yes, it is possible to add programmatically several people (some of the demos show 2 or more people together). As a workflow, I'd suggest the following:

1. Create the postures in the editor
2. For each posture use [Get posture] button to get an array of numbers, that describes the posture
3. In your program create several people and assign a posture (from step 2) to each of them
4. In your program change the body position in space and the body rotation in space

If you need more details, let me know.

Here is an example of two bodies:
[https://boytchev.github.io/mannequin.js/examples/example-posture-blend.html](https://boytchev.github.io/mannequin.js/examples/example-posture-blend.html)

Pavel
